tc/core/halide2isl.cc

```diff
@@ -197,10 +199,9 @@ std::vector<isl::aff> makeIslAffBoundsFromExpr(
   } else if (const Div* op = e.as<Div>()) {
     return combineSingleAffs(space, op, &isl::aff::div);
   } else if (const Mod* op = e.as<Mod>()) {
-    std::vector<isl::aff> result;
     // We cannot span multiple constraints if a modulo operation is involved.
     // x > max(a,b) % C is not equivalent to (x > a % C && x > b % C).
-    auto lhs = makeIslAffBoundsFromExpr(space, e, false, false);
+    auto lhs = makeIslAffBoundsFromExpr(space, op->a, false, false);
```

Comment:
using `e` would lead to infinite recursion and hence segfault

Reply:
This comment should be in the commit message.